### DISCUSSION TITLE
vcpkg vscode installation error

### DISCUSSION THREAD

**Discussion by viktorivan** [alandefreitas/matplotplusplus#374] [Q&A]
Hi, I have installed matplot++ through vcpkg, and verified it.

$ vcpkg search matplotplusplus
matplotplusplus          1.2.0#1          A C++ graphics library for data visualization
matplotplusplus[blas]                     BLAS support for Matplot++
matplotplusplus[fftw]                     fftw3 support for Matplot++
matplotplusplus[jpeg]                     JPEG support for Matplot++
matplotplusplus[lapack]                   LAPACK support for Matplot++
matplotplusplus[opencv]                   opencv support for Matplot++
matplotplusplus[opengl]                   OpenGL backend for Matplot++
matplotplusplus[zlib]                     ZLIB and libpng support for Matplot++
The result may be outdated. Run `git pull` to get the latest results.
If your port is not listed, please open an issue at and/or consider making a pull request.  -  https://github.com/Microsoft/vcpkg/issues

but, when I am running the examples I have the following error:

fatal error: matplot/matplot.h: No such file or directory
    2 | #include <matplot/matplot.h>
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.

I am using LinuxMint 21.2 and VSCode

**Comment by alandefreitas** [comment]
Search performs a case-insensitive search through all available package names and descriptions. Not the packages you have installed. And once the package is installed, you still have to integrate vcpkg in your workflow (usually via cmake), and call find_package(...) or whatever vcpkg recommends.
